test/Microsoft.AspNetCore.Server.HttpSys.FunctionalTests/Listener/ResponseCachingTests.cs

```diff
@@ -537,6 +537,7 @@ public async Task Caching_SendFileWithFullContentLength_Cached()
         }
 
         [ConditionalFact]
+        [OSSkipCondition(OperatingSystems.Windows, WindowsVersions.Win7, WindowsVersions.Win2008R2, SkipReason = "Flaky HttpClient on NT 6.1")]
```

Comment:
FYI - there have been a lot of issues in the HttpClient since 2.0. I wonder if this correlated. Have you been able to identify what the HttpClient is doing wrong? If so, it would be good to talk to the corefx team about it.

Reply:
Issue link?